docs/README.md

```diff
@@ -0,0 +1,91 @@
+# Spectacle Documentation Site
+
+Now built with react-static!
+
+## Getting Started
+
+To install and run the docs site locally:
+
+```sh
+cd docs
+yarn install
+yarn start
+```
+
+Note that paths in local development are based on a root of "/" but be careful when defining relative and absolute paths
+inline or doing url parsing, as the production output root will be "open-source/spectacle."
+
+## Want to see if you're ready to :shipit:?
+
+To build the staging build output and serve it with the canonical path it'll have when built as a lander for formidable.com:
+
+```sh
+cd docs
+yarn stage
+yarn serve
+```
+
+This step is important for validating that both the `basePath` used by the static HTML output and the `basename` used
+by the client-side router are working as expected. This is also where you'll want to validate that there are no hardcoded, inlined, or malformed asset paths that worked locally but will not resolve correctly in production!
+
+<!--
+## Ready to Publish?
```

Comment:
As I imagine these steps are still going to change, I've only commented out this section instead of opting to gut it completely. Pending deployment/publishing changes, I will update this section as we see fit